static/sass/_instance_list.scss

```diff
@@ -0,0 +1,42 @@
+.p-instance-table {
+  @media screen and (max-width: 1400px) {
+    th,
+    td {
+      &:nth-child(4),
+      &:nth-child(5) {
+        display: none;
+      }
+    }
+  }
+}
+
+.p-instance-table-with-panel {
+  th,
+  td {
+    &:nth-child(4),
+    &:nth-child(5) {
+      display: none;
+    }
+  }
+}
+
+.u-row:hover,
+.u-row-selected {
+  background-color: $colors--light-theme--background-hover;
+}
+
+.u-instance-actions {
+  visibility: hidden;
+}
+
+.u-row:hover {
+  .u-instance-actions {
+    visibility: visible;
+  }
+}
+
+.u-row-selected:hover {
+  .u-instance-actions {
+    display: none;
```

Comment:
This is a little wired. I would also show them on the selected row on hover.

Reply:
I don't see why: we have the actions in the side panel, for the selected row.

Reply:
I think it would be better for consistency.

Reply:
Let's keep this point open and ask design input about it.